src/utils/utils.jl

```diff
@@ -28,6 +32,28 @@ function clear_serialized_system(name::String)
     return
 end
 
+function clear_serialized_system(
+    name::String,
+    case_args::Dict{Symbol, <:Any} = Dict{Symbol, Any}(),
+)
+    file_path = get_serialized_filepath(name, case_args)
+
+    try
+        if isfile(file_path)
+            @debug "Deleting file at " file_path
+            rm(file_path; force = true)
+        end
+    catch e
+        @show uperm(dirname(file_path))
```

Comment:
Have you reproduced the issue with these debugging statements? Did they help? If not, should they be removed?

Reply:
Yes, the catch blocks works, confirming that we can access the directory. 
I don't think we need to show all the information about the directory to the user but perhaps we can keep the part where the list of files in the directory are displayed.

Reply:
What value is that providing? We have confirmed that the directory is readable and so that isn’t the issue.